integration-tests/features/component_analysis_v2_smoke_tests.feature

```diff
@@ -0,0 +1,121 @@
+     | npm        | react-dom           | 16.8.6               | #13
+     | npm        | mkdirp              | 0.5.1                | #14
+     | npm        | prop-types          | 15.7.2               | #15
+     | npm        | colors              | 1.3.3                | #16
+     | npm        | glob                | 7.1.4                | #17
+     | npm        | body-parser         | 1.19.0               | #18
+     | npm        | yargs               | 13.3.0               | #19
+     | npm        | minimist            | 1.2.0                | #20
+
+
+  Scenario Outline:  Check the component analysis V2 REST API endpoint for 20 most popular PyPi components
+    Given System is running
+    Given Three scale preview service is running
+        
+        When I acquire the use_key for 3scale
+        Then I should get the proper user_key
+        
+        When I start v2 component analyses <ecosystem>/<package>/<version> with user_key
+        Then I should get 200 status code
+            And I should receive a valid JSON response
+
+        When I look at the component analysis duration
+        Then I should see that the component analysis duration is less than 10 second
+      
+        Then I should get no result or recommendation with report
+
+         Examples: EPV
+        | ecosystem  | package             | version              |
+        | pypi       | pip                 | 19.2.1               | #1
+        | pypi       | urllib3             | 1.25.3               | #2
+        | pypi       | botocore            | 1.12.202             | #3
+        | pypi       | six                 | 1.12.0               | #4
+        | pypi       | python-dateutil     | 2.8.0                | #5
+        | pypi       | s3transfer          | 0.2.1                | #6
+        | pypi       | pyyaml              | 5.1.2                | #7
+        | pypi       | docutils            | 0.15.2               | #8
+        | pypi       | pyasn1              | 0.4.6                | #9
+        | pypi       | requests            | 2.22.0               | #10
+        | pypi       | setuptools          | 41.0.1               | #11
+        | pypi       | jmespath            | 0.9.4                | #12
+        | pypi       | certifi             | 2019.6.16            | #13
+        | pypi       | awscli              | 1.16.212             | #14
+        | pypi       | rsa                 | 4.0                  | #15
+        | pypi       | futures             | 3.3.0                | #16
+        | pypi       | idna                | 2.8                  | #17
+        | pypi       | colorama            | 0.4.1                | #18
+        | pypi       | wheel               | 0.33.4               | #19
+        | pypi       | chardet             | 3.0.4                | #20
+    
+
+   Scenario Outline: Check the component analysis V2 REST API endpoint for 20 most popular maven components
```

Comment:
i think we should add these cases after we actually do user integration , because if we add now and it will be a negative case and after when we actually add the user registration it will fail ?

Reply:
I think we need a test to make sure we are not exposing any fields that we are not supposed to.

Reply:
okay will add

Reply:
added the check for if any registered fields are exposed